### PR TITLE
Created Angular form that submits POST request to backend

### PR DESCRIPTION
**Pull request by shubha-rajan**
First of a few (probably like 3?) PRs towards closing #2. 
Sticker URLs are hard-coded for now, until we get the code for storage set up.
Also pls admire the cute stickers I chose.

### CODE REVIEW

File: frontend/src/app/message.service.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import { Injectable } from '@angular/core';
+import { HttpClient, HttpHeaders } from '@angular/common/http';
+
+@Injectable()
+export class MessageService {
+    constructor(private http: HttpClient) { }
+
+    baseUrl = 'http://localhost:8000';
```
Comment (murog) [inline frontend/src/app/message.service.ts]: can you make this an environment variable?

File: frontend/src/app/message-display/message-display.component.ts [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+@Component({
+  selector: 'app-message-display',
+  templateUrl: './message-display.component.html',
+  styleUrls: ['./message-display.component.scss'],
+  providers:[ MessageService ]
+})
+export class MessageDisplayComponent implements OnInit{
+
+  constructor(private messageService: MessageService) { }
+  // hard-coded for now
```
Comment (murog) [inline frontend/src/app/message-display/message-display.component.ts]: nit: add "TODO:"

File: frontend/src/app/message-display/message-display.component.ts [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+  messages  = [
+    {
+      name: "Crisco",
+      body: "Northern pikas are most active and mostly feed soon after dawn and as dusk approaches.",
+      stickerUrl: "https://i.pinimg.com/originals/63/b3/49/63b349f74f7f2e498e1ca74c66b829fa.jpg",
+      timestamp: "11-19-2019"
+  }
+  ]
+
+  ngOnInit() {
```
Comment (murog) [inline frontend/src/app/message-display/message-display.component.ts]: nit: add TODO:

File: frontend/src/app/message-form/message-form.component.ts [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+  onSubmit() { 
+    this.messageService.postMessage(this.model)
+    .subscribe(()=>{
+      this.newMessage();
+    })
+  }
+  constructor(private messageService: MessageService) { }
+
+  ngOnInit() {
+    // this.messageService.getStickers()
```
Comment (murog) [inline frontend/src/app/message-form/message-form.component.ts]: nit: add TODO:

File: frontend/src/app/message.spec.ts [hunk 5]
```diff
@@ -0,0 +1,12 @@
+import { Message } from './message';
+
+describe('Message', () => {
+  it('should create an instance', () => {
+    expect(new Message("He-Man","Heyayayaya")).toBeTruthy();
```
Comment (murog) [inline frontend/src/app/message.spec.ts]: love it

File: frontend/src/app/message.service.ts (hunk 1 above)
Comment (shubha-rajan) [reply frontend/src/app/message.service.ts]: done!

File: frontend/src/app/message-display/message-display.component.ts (hunk 2 above)
Comment (shubha-rajan) [reply frontend/src/app/message-display/message-display.component.ts]: done

File: frontend/src/app/message-display/message-display.component.ts (hunk 3 above)
Comment (shubha-rajan) [reply frontend/src/app/message-display/message-display.component.ts]: done

File: frontend/src/app/message-form/message-form.component.ts (hunk 4 above)
Comment (shubha-rajan) [reply frontend/src/app/message-form/message-form.component.ts]: done

**Comment by shubha-rajan** [comment]
Update: I ended up removing the example.env file and just following the angular convention of using `environment.ts` config files. dotenv (and pretty much anything with an fs dependency) on the front end is a pain. Besides, it's not like we have any API keys or anything.

Also, I moved the logic for getting messages to the app component, and passed a callback to refresh messages to the form component. That way when it's all working, you'll see your message as soon as you post it. :)

**Pull request merged**